src/index.css

```diff
@@ -0,0 +1,116 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  color: white;
+  background-color: rgb(184, 41, 124);
+}
+
+.navBar {
+  width: 100%;
+  padding: 18px;
+  display: flex;
+  flex-direction: row;
+  gap: 40%;
+  border-bottom: 1px solid rgb(0, 0, 0);
+  color: black;
+}
```

Comment:
- [ ] [OPTIONAL] I would suggest to use the color white or something that you have used for the other texts on your page, as currently this part is looking a bit odd in the page :point_down: 
![image](https://user-images.githubusercontent.com/97521557/219846686-b88f576d-4dab-463f-a7de-a223533395b7.png)